src/azul/indexer/document.py

```diff
@@ -340,6 +345,29 @@ def from_index(self, value: Number) -> Optional[bool]:
 
 null_bool: NullableBool = NullableBool()
 
+
+class NullableDateTime(FieldType[Optional[datetime], str]):
+    null_date = '0001-01-01T00:00:00.000000Z'
```

Comment:
This should be derived from the canonical form i.e. a datetime instance, using the format below.

Reply:
```suggestion
    null = '0001-01-01T00:00:00.000000Z'
```